src/main/java/org/fcrepo/spec/testsuite/crud/ExternalBinaryContent.java

```diff
@@ -97,6 +238,33 @@ public void putCreateExternalBinaryContentCopy(final String uri) {
                         "request entity, as indicated by a link with " +
                         "rel=\"http://fedora.info/definitions/fcrepo#ExternalContent\"",
                 "https://fcrepo.github.io/fcrepo-specification/#external-content", ps);
+
+        final String location = joinLocation(uri, info.getId());
```

Comment:
hmm... since we are talking about the URI passed in from `param1`, which is used in all of the tests, I would suggest that ensuring the `uri` ends in `/` be handled once, here: https://github.com/fcrepo/Fedora-API-Test-Suite/blob/master/src/main/java/org/fcrepo/spec/testsuite/TestSuiteGlobals.java#L58
..which is invoked here: https://github.com/fcrepo/Fedora-API-Test-Suite/blob/master/src/main/java/org/fcrepo/spec/testsuite/App.java#L101